src/components/TextareaWithUploader.vue

```diff
@@ -119,8 +119,8 @@ const handleError = (error: any) => {
   .el-button {
     font-weight: normal;
     font-size: 14px;
-    background-color: #f7f7f7;
-    border-color: #f7f7f7;
+    background-color: --color-bg-primary;
+    border-color: --color-border-primary;
```

Comment:
Don't we need to use the `var` function here??

```css
background-color: var(--color-bg-primary);
```

Reply:
oh yes!

Reply:
fixed it, PTAL

Reply:
cool